libs/wingsdk/src/target-sim/domain.ts

```diff
@@ -16,4 +17,13 @@ export class Domain extends cloud.Domain {
   public _toInflight(): string {
     return makeSimulatorJsClient(__filename, this);
   }
+
+  public toSimulator(): BaseResourceSchema {
+    return {
+      type: cloud.DOMAIN_FQN,
+      path: this.node.path,
+      props: {},
+      attrs: {},
+    };
+  }
```

Comment:
Domain isn't currently supported in the sim, right?  Should this throw or something?

Reply:
I wasn't sure - I see we have an open issue for adding support [here](https://github.com/winglang/wing/issues/4556), but it looks like the class was already added to the `target-sim` folder, and we already have docs about the simulator implementation on our [docs](https://www.winglang.io/docs/standard-library/cloud/domain#target-specific-details). @marciocadev should this be removed?

Reply:
In any case it doesn't seem like `Domain` has any inflight methods, so I'm okay leaving in some of this boilerplate for now for